### PR TITLE
Updating for release 1.9

### PR DESCRIPTION
**Pull request by moxiegirl**
@SvenDowideit This is what we used for 1.9 today.  Might as well update

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

File: all-projects.yml [hunk 1]
```diff
@@ -36,20 +36,26 @@ projects:
 
     - name: kitematic
       org: kitematic
+      ref: master
```
Comment (SvenDowideit) [inline all-projects.yml]: don't need this here (or the others where its `master`) - it comes from the `defaults`

**Comment by SvenDowideit** [comment]
Does that mean that most of the projects are no longer using a `docs` branch for tracking intermediate docs updates?

**Comment by moxiegirl** [comment]
@SvenDowideit No, it means they aren't doing the procedure to set to `docs` until after the release.

**Comment by SvenDowideit** [comment]
k, LGTM

**Pull request merged**